react/features/connection-indicator/components/native/ConnectionIndicator.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import React from 'react';
+
+import { BaseIndicator } from '../../../base/react';
+import { connect } from '../../../base/redux';
+
+import AbstractConnectionIndicator, {
+    type Props,
+    type State
+} from '../AbstractConnectionIndicator';
+
+import { CONNECTOR_INDICATOR_COLORS } from './styles';
+
+/**
+ * Implements ain indicator to show the quality of the connection of a participant.
```

Comment:
I know a person named Ain. He'll be happy to know his indicator is being implemented.

Reply:
Yeah, when we were kids, I promised him I'll implement it once I get to work for a company like this. I always keep my promises.